### PR TITLE
Fixes #12154: Create a generic method to define environment variable

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/12154

### CODE REVIEW

**Comment by ncharles** [comment]
There's an issue with this solution: /etc/environment is not reloaded automatically by cf-execd (which is the process runnin cf-agent). So it won't be known by subsequent agent run, unless cf-execd is restarted

File: tree/30_generic_methods/environment_variable.cf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# @description Enforce an environment variable value
+#
+# @parameter key Name of the environment variable
+# @parameter value Value of the environment variable
+# 
+# @class_prefix environment_variable
+# @class_parameter key
+# This bundle will define a class environment_variable_${key}_{kept,not_ok}
+
+bundle agent environment_variable(key, value)
```
Comment (amousset) [inline tree/30_generic_methods/environment_variable.cf]: environment_variable_present?

Comment (amousset) [reply tree/30_generic_methods/environment_variable.cf]: s/key/name/

**Comment by peckpeck** [comment]
ping @ncharles

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**